src/keys.rs

```diff
@@ -334,6 +334,15 @@ impl From<FullViewingKey> for SpendValidatingKey {
 }
 
 impl FullViewingKey {
+    /// Create FVK from the given sk and ak.
+    pub fn from_sk_ak(sk: &SpendingKey, ak: SpendValidatingKey) -> Self {
```

Comment:
The documentation here is very insufficient:
- It makes no mention of how `sk` and `ak` are checked or combined (in particular, that you get a different FVK than if you just go from `sk`).
- It makes no mention of FROST (or why you would want to use this API).
- It is not behind a feature flag so is exposing a normative public API which I do not want to be present without a corresponding specification (which will eventually be ZIP 312 once https://github.com/zcash/zips/pull/883 is finished).

Also, not documented as an addition in the changelog.